airbyte-commons-worker/src/main/java/io/airbyte/workers/process/Metadata.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.process;
+
+/**
+ * The following variables help, either via names or labels, add metadata to processes actually
+ * running operations to ease operations.
+ */
+public class Metadata {
```

Comment:
Consider marking this class as `final` with a private constructor to avoid accidental instantiation or inheritance.  Another option would be to go the sealed class route and not permit any direct subclasses.

Reply:
oh good idea!